app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -47,6 +47,10 @@ def supports_port?
     true
   end
 
+  def supports_refresh_new_target?
+    true
+  end
```

Comment:
So the problem is you aren't using the mixin you're just defining the method directly.
All you have to do is put `supports :refresh_new_target` like this https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/supports_feature_mixin.rb#L7